docs/design-language.md

```diff
@@ -223,6 +223,22 @@ A few guidelines should be followed to ensure that the list element looks unifor
 
 When a hover state is triggered, it cannot just rely on a colour change. There also must be a state change that shows the differentiation. For example, in our programs, there is a background colour change, as well as the addition of a side border colour. Hover states always include a styling change, and the background colour will get a gradient.
 
+## Loading
+
+Two options are available to denote a loading state, the "3 dot loader" and "Skeleton". Skeleton is the preferred method with the 3 dots being phased out where applicable.
```

Comment:
I don't think we are "phasing out" the three dot loader. It's all about the appropriate use case for each and the three dot loader is still the better choice in many scenarios (which should be described in more detail here).

Reply:
Updated.

Reply:
Better, Thanks